morello/morello.md

```diff
@@ -0,0 +1,287 @@
+# Morello Supplement to the Arm C Language Extensions
+
+::: version
+01alpha
+:::
+
+::: issued
+Date of Issue: 02 July 2021
```

Comment:
Please change this to `Date of Issue: TBD`. Please do the same for the MVE file.

Reply:
For the MVE file, do I change the entry in the "Document history" section? If so, would I have to shorten "Development version based on 01alpha" due to the entries in the "Change" column being around 20 characters at most?